Localization/en-US/Mods.PathOfTerraria.Misc.hjson

```diff
@@ -30,3 +30,11 @@ FishronSpawn: You hear a roar in the distance...
 ImpendingDoom: A pulse quickens at the center of the world...
 VoidPearlCondition: In hardmode or during the Unleashing Chaos quest
 SubworldItemDisabler: You cannot use this item in a subworld.
+Damage: damage
```

Comment:
Too bound to English. This needs to be in `{0} damage` format, languages need to be able to define order as well as casing for the number in context.